src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/ClusterMonitorActor.scala

```diff
@@ -229,6 +229,9 @@ class ClusterMonitorActor(val cluster: Cluster,
     logger.info(s"Cluster ${cluster.projectNameString} has been deleted.")
 
     for {
+      // delete the init bucket so we don't continue to accrue costs after cluster is deleted
+      _ <- deleteInitBucket
```

Comment:
how convenient that this code was left behind 😄

Reply:
Heh, yeah it was a complete oversight that we missed deleting this bucket at cluster deletion time